modules/update-service-mirror-release.adoc

```diff
@@ -146,7 +146,7 @@ $ TEMP_REPOSITORY='<temp_repository_name>'
 +
 For `<temp_repository_name>`, specify the name of a temporary repository to create in your
 registry, such as `ocp4/temp`.
-... Take the media to the restricted network environment and upload the images to the temporary repository in the local container registry. Uploading the images to the temporary repository allows you to then mirror them from there to the final repositories at which time the release images will be mirrored to a separate repository.
+... Take the media to the restricted network environment and upload the images to the temporary repository in the local container registry. By uploading the images to the temporary repository, you can then mirror them to the final repositories so that the release images are mirrored to a separate repository.
 +
 [source,terminal]
 ----
```

Comment:
Do you mind separating these two commands into separate code blocks?

Reply:
Resolved